src/shadowbox/server/server_access_key.ts

```diff
@@ -196,6 +196,16 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
     } else {
       id = this.generateId();
     }
+
+    const accessKeys = this.listAccessKeys();
+    const isPasswordConflict = accessKeys.some(
+      (accessKey) => accessKey.proxyParams.password == params?.password
+    );
+
+    if (isPasswordConflict) {
+      throw new errors.PasswordConflict();
+    }
```

Comment:
Just making it more compact and grouping the password check logic.:
```suggestion
    const isPasswordConflict = this.listAccessKeys().some(
      (accessKey) => accessKey.proxyParams.password == params?.password
    );
    if (isPasswordConflict) {
      throw new errors.PasswordConflict();
    }
```